front50-oracle-bmcs/src/main/java/com/netflix/spinnaker/front50/config/OracleBMCSProperties.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2017 Oracle America, Inc.
+ *
+ * The contents of this file are subject to the Apache License Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * If a copy of the Apache License Version 2.0 was not distributed with this file,
+ * You can obtain one at https://www.apache.org/licenses/LICENSE-2.0.html
+ */
+package com.netflix.spinnaker.front50.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("spinnaker.oraclebmcs")
+public class OracleBMCSProperties {
+
+  private String bucketName = "_spinnaker_front50_data";
```

Comment:
I'm not sure i fully understand. However, the Oracle BMCS Object Storage Service is a regional service, not global and the buckets names need only be unique within the tenancy/account. Does that help?